public/components/Notifications/NotificationForm.tsx

```diff
@@ -37,10 +38,12 @@ export interface NotificationFormProps {
   onMessageBodyChange: (message: string) => void;
   onMessageSubjectChange: (subject: string) => void;
   onNotificationToggle?: (enabled: boolean) => void;
+  context: any
```

Comment:
Let's avoid using `any` when possible, so add the correct type for this

Reply:
Created interface for TriggerContext. TriggerContext could be enhanced as needed later.